lib/browser/guest-view-manager.js

```diff
@@ -340,35 +340,35 @@ const isWebViewTagEnabled = function (contents) {
   return isWebViewTagEnabledCache.get(contents)
 }
 
-const handleMessage = function (channel, handler) {
-  ipcMainInternal.on(channel, (event, ...args) => {
-    if (isWebViewTagEnabled(event.sender)) {
-      handler(event, ...args)
-    } else {
-      event.returnValue = null
-    }
-  })
+const assertWebViewTagEnabled = function (contents) {
+  if (!isWebViewTagEnabled(contents)) {
+    throw new Error('<webview> disabled')
```

Comment:
Hmm, is this code only reached when we receive a webview message from a process that shouldn't be sending any webview messages? If that's the case, we should make this message clearer, e.g.
```suggestion
    throw new Error(`<webview> IPC method ${ipc_method_name} called on WebContents with webview disabled (${webcontents_id})`)
```

Additionally, we should probably unceremoniously tear down that process because it's doing something it shouldn't be.

Reply:
the exception is propagated back to the renderer. I've added a `console.error` with a more detailed message similar to the suggested one.

Reply:
> Additionally, we should probably unceremoniously tear down that process because it's doing something it shouldn't be.
That I would do in a follow up PR, I just wanted to adopt the IPC helpers in this one